brillig/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![warn(unused_crate_dependencies)]
+#![warn(unreachable_pub)]
+
+//! The Brillig bytecode is distinct from regular [ACIR][acir] in that it does not generate constraints.
+//! This is a generalization over the fixed directives that exists within in the ACVM.
```

Comment:
Linking https://github.com/noir-lang/acvm/issues/395, as we will need to update this sentence, once that is completed